src/PostgREST/Private/QueryFragment.hs

```diff
@@ -87,9 +87,9 @@ asBinaryF fieldName = "coalesce(string_agg(_postgrest_t." <> pgFmtIdent fieldNam
 locationF :: [Text] -> SqlFragment
 locationF pKeys = [qc|(
   WITH data AS (SELECT row_to_json(_) AS row FROM {sourceCTEName} AS _ LIMIT 1)
-  SELECT array_agg(json_data.key || '=' || coalesce('eq.' || json_data.value, 'is.null'))
+  SELECT array_agg(json_data.key || '=eq.' || json_data.value)
   FROM data CROSS JOIN json_each_text(data.row) AS json_data
-  {("WHERE json_data.key IN ('" <> intercalate "','" pKeys <> "')") `emptyOnFalse` null pKeys}
+  WHERE json_data.key IN ('{intercalate "','" pKeys}')
```

Comment:
More verbose but more clear I think:
```suggestion
  WHERE json_data.key IN ('{fmtPKeys}')
)|]
  where fmtPKeys = intercalate "','" pKeys
```
I tried the `= ANY ARRAY` form but didn't work somehow.